src/FormField/FormField.js

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import Flex from '../Flex';
 import Label from '../Label';
 import Paragraph from '../Paragraph';
 import StyledFormField from './StyledFormField';
 
-const FormField = ({ caption, children, label, id }) => (
+const FormField = ({ caption, children, label, labelAside, id }) => (
   <StyledFormField caption={caption}>
-    {label && <Label htmlFor={id}>{label}</Label>}
+    <Flex alignItems="center">
```

Comment:
My "flex" skills are a bit lacking, what's the purpose of the dual flexes with alignItems center here?

Reply:
The first one is the container, and the second is just so we can set `flex="1"`